examples/evaluator/golang/simple/evaluate/evaluator.go

```diff
@@ -11,14 +11,48 @@
 // WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
 // See the License for the specific language governing permissions and
 // limitations under the License.
+
 package evaluate
 
 import (
+	"sort"
+	"strings"
+
+	"open-match.dev/open-match/examples"
 	harness "open-match.dev/open-match/pkg/harness/evaluator/golang"
 	"open-match.dev/open-match/pkg/pb"
 )
 
 // Evaluate is where your custom evaluation logic lives.
+// This sample evaluator sorts and deduplicates the input matches.
 func Evaluate(p *harness.EvaluatorParams) ([]*pb.Match, error) {
-	return p.Matches, nil
+	// Sort in descending numerical order
+	scoreBy := func(a, b *pb.Match) bool {
```

Comment:
`scoreBy` is not very descriptive. `scoreInDescendingOrder`

That way it reads `by(scoreInDescendingOrder).Sort(p.Matches)` which is still awkward but more descriptive.